hack/ci/windows.ps1

```diff
@@ -994,4 +994,14 @@ Finally {
     Nuke-Everything
     $Dur=New-TimeSpan -Start $StartTime -End $(Get-Date)
     Write-Host -ForegroundColor $FinallyColour "`nINFO: executeCI.ps1 exiting at $(date). Duration $dur`n"
+
+    # Temporary solution to avoid disk to fill on RS5
+    if ($env:COMPUTERNAME -match "jenkins-rs5-") {
+		if ((Get-Volume -DriveLetter D).SizeRemaining -lt 15GB) {
+			Write-Host -ForegroundColor Green "INFO: D -drive is almost full, rebooting..."
+			shutdown -r -t 15
```

Comment:
Nit: This is fine but could use the `Restart-Computer` PowerShell command, though the delay would be done separately, e.g. `Start-Sleep 15; Restart-Computer`, if waiting is important.